.npmrc

```diff
@@ -1,4 +0,0 @@
-//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
```

Comment:
Should we ignore this file in git?

Reply:
I'm not sure, there may be something to add to it in the future. But we could always remove it from the ignore if that becomes the case